content/cybersecurity/concepts/nmap/terms/udp-scan/udp-scan.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'UDP Scan'
+Description: 'Scans (only) the UDP Ports of a target, which are not scanned in a regular scan. Can be combined with any TCP scan'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+An **UDP Scan** scans (only) the UDP Ports of a target, while the [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan) only scans the TCP Ports. It is performed by using the `-sU` option and can be combined with any TCP scan type, for example SYN scan (`-sS`) to scan UDP and TCP Ports in the same run. UDP Scan is normally slower and more difficult than a TCP scan.
```

Comment:
```suggestion
A **user diagram protocol (UDP) scan** checks for any UDP ports that are deployed on a target. Conversely, the [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan) only scans the TCP ports. UDP scans are normally slower and more difficult than TCP scans.
```

I think the second sentence makes more sense somewhere in the "Syntax" section below.